### ISSUE TITLE
K8SSAND-1493 ⁃ Unable to deploy k8ssandra-cluster-a /k8ssandra-cluster-b helm unable to deploy

### ISSUE THREAD

**Issue by wskibum** [k8ssandra/k8ssandra#1400]
Ubuntu 20.04
Kubernetes 1.20.14
helm version
version.BuildInfo{Version:"v3.8.2"

helm repo add k8ssandra https://helm.k8ssandra.io/
helm repo update

helm repo update
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "jetstack" chart repository
...Successfully got an update from the "k8ssandra" chart repository
...Successfully got an update from the "traefik" chart repository
...Successfully got an update from the "bitnami" chart repository
Update Complete. ⎈Happy Helming!⎈

helm install k8ssandra-cluster-a  k8ssandra/kasandra-cluster  --set cassandra.cassandraLibDirVolume.storageClass=csi-rbd-sc
**Error: INSTALLATION FAILED: failed to download "k8ssandra/kasandra-cluster"**

┆Issue is synchronized with this [Jira Task](https://k8ssandra.atlassian.net/browse/K8SSAND-1493) by [Unito](https://www.unito.io)
┆friendlyId: K8SSAND-1493
┆priority: Medium

**Comment by wskibum** [comment]
work a round solution which may be better than the cluster install. I created a value.yaml and deployed a "Data Center" instead. This is a lab for performance testing so I don't think it really matters and I was able to get everything I wanted installed with it. Really wanted medusa for s3 backups and was able to add that. 

cat k8ssandra.yaml 

{code} 

cassandra: 
version: "3.11.10" 
cassandraLibDirVolume: 
storageClass: csi-rbd-sc 
size: 5Gi 
allowMultipleNodesPerWorker: true 
heap: 
size: 1G 
newGenSize: 1G 
resources: 
requests: 
cpu: 1000m 
memory: 2Gi 
limits: 
cpu: 1000m 
memory: 2Gi 
datacenters: 
- name: dc1 
size: 6 
medusa: 
enabled: true 
bucketName: k8ssandra-bucket-dev 
storageSecret: medusa-bucket-key 
storage: s3 
storage_properties: 
region: ca-central-1 
racks: 
- name: default 
kube-prometheus-stack: 
grafana: 
adminUser: admin 
adminPassword: admin123 
stargate: 
enabled: true 
replicas: 1 
heapMB: 256 
cpuReqMillicores: 200 
cpuLimMillicores: 1000 
{code} 

helm install -f k8ssandra.yaml k8ssandra k8ssandra/k8ssandra
